assets/js/zzzz-search-data.json

```diff
@@ -0,0 +1,58 @@
+---
```

Comment:
@SgtSilvio whats up with renaming this file with `zzzz` ?

Reply:
Jekyll evaluates files in alphabetical order.
So before when it was called `search-data.json` pages that started with a letter `t` or alphabetically greater were not rendered to html yet, so their plain md was included in the `search-data.json`

Reply:
Woah TIL! Nice hack 😄

Reply:
I remember that I found this out by trial and error and I was so confused that some pages were html and some still md.